api/src/certification/session/domain/usecases/add-certification-candidate-to-session.js

```diff
@@ -1,3 +1,7 @@
+/**
+ * @typedef {import ('../../../shared/domain/usecases/index.js').dependencies} deps
```

Comment:
Je trouve cette PR très habile, merci. J'ai une interrogation : je n'ai pas réussi à trouver dans la doc de JSDoc qu'on pouvait utiliser des imports comme ceci. Manifestement cela fonctionne 😄, mais comment et où as-tu eu cette idée d'utiliser ainsi des imports STP ?

Reply:
tu l'expliqueras lors de ta MEJ @Steph0 ? 🤭

Reply:
@lego-technix alors, j'ai retouvé un des liens qui m'ont servis pour ce qui est de l'import : https://github.com/jsdoc/jsdoc/issues/1537

Du coup comme on le lira, ça tire partie de la version TypeScript de la JSDoc, qui est supporté out-of-the-box par les IDE. La documentation se trouve alors là : https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html

Il y a une suggestion pour une implémentation "pure" JSDOC dans l'issue, qui pointe ici : https://stackoverflow.com/a/55767692

Je n'ai pas tenté la version pure JSDoc du coup, car la version import me convenait bien et que ça apportait la flexibilité de le nommage et l'aliasing des dépendances dans le fichier d'injection des dépendances.

Reply:
Bien reçu @Steph0 merci beaucoup ! Cette explication mériterait d'être dans la section _Proposition_ de cette PR.

Reply:
Et on pourra alors marquer cette discussion comme _Resolved_.

Reply:
Entendu, c'est ajouté !